README.md

```diff
@@ -19,6 +19,9 @@ An introduction to the NVIDIA Container Runtime is also covered in our [blog pos
 **If you have a custom `/etc/docker/daemon.json`, the `nvidia-docker2` package might override it.**  
 
 #### Ubuntu 14.04/16.04/18.04, Debian Jessie/Stretch
+
+The Ubuntu linux will install `docker.io` by default which doesn't install the latest version of docker meaning that you need to pin the nvidia-docker version when trying to install nvidia-docker: <https://github.com/NVIDIA/nvidia-docker/wiki/Frequently-Asked-Questions#how-do-i-install-20-if-im-not-using-the-latest-docker-version>
```

Comment:
Reword to "Ubuntu Linux will install by default ... which isn't the latest version of docker. This implies that you will need to pin the version of NVIDIA docker. [See more information here](https://github.com/NVIDIA/nvidia-docker/wiki/Frequently-Asked-Questions#how-do-i-install-20-if-im-not-using-the-latest-docker-version).

Reply:
Doen.